Makefile

```diff
@@ -595,19 +595,23 @@ help: ## Display this help.
 ##@ Linting:
 
 .PHONY: lint
-lint: $(GOLANGCI_LINT) lint-latest ## Lint codebase.
-	$(GOLANGCI_LINT) run -v --print-resources-usage --go=$(GO_VERSION) $(GOLANGCI_LINT_EXTRA_ARGS)
+lint: $(GOLANGCI_LINT) lint-azure-latest ## Lint the codebase.
+	$(GOLANGCI_LINT) run -v $(GOLANGCI_LINT_EXTRA_ARGS)
+
+.PHONY: lint-fast
+lint-fast: $(GOLANGCI_LINT) ## Lint the codebase with fast linters only.
+	GOLANGCI_LINT_EXTRA_ARGS+=--fast-only $(MAKE) lint
 
 .PHONY: lint-fix
 lint-fix: $(GOLANGCI_LINT) ## Lint the codebase and run auto-fixers if supported by the linter.
-	GOLANGCI_LINT_EXTRA_ARGS=--fix $(MAKE) lint
+	GOLANGCI_LINT_EXTRA_ARGS+=--fix $(MAKE) lint
 
-lint-full: $(GOLANGCI_LINT) ## Run slower linters to detect possible issues.
-	$(GOLANGCI_LINT) run -v --fast=false --go=$(GO_VERSION)
+.PHONY: lint-full
+lint-full: $(GOLANGCI_LINT) lint ## Lint the codebase.
 
-.PHONY: lint-latest ## TODO: update the lint-latest to lint-azure-latest
-lint-latest:
-	./hack/lint-latest.sh ## TODO: update the lint-latest.sh to lint-azure-latest.sh
+.PHONY: lint-azure-latest ## Check for usage of the "latest" floating Azure API version.
```

Comment:
Thank you!

Reply:
I had been ignoring this `TODO` that gets printed every time you lint, but once I looked at it I understood what @devigned meant.